pleskdistup/common/src/leapp_configs.py

```diff
@@ -238,13 +238,19 @@ def add_repositories_mapping(repofiles: typing.List[str], ignore: typing.List =
 
 def set_package_repository(package: str, repository: str, leapp_pkgs_conf_path: str = LEAPP_PKGS_CONF_PATH) -> None:
     pkg_mapping = None
+    log.debug("Reconfigure mapping for package '{}' to repository '{}'".format(package, repository))
```

Comment:
I suggest to use f-strings or at least named placeholders as above instead of `{}`.